.travis.yml

```diff
@@ -19,9 +19,15 @@ python:
   - 3.6
 services:
   - docker
+before_install:
+  - if [[ "$TRAVIS_BRANCH" == "master" ]]; then
+      VERSION=latest;
+    else
+      VERSION=$TRAVIS_BRANCH;
+    fi
 install:
-  - docker build -t max-image-caption-generator .
-  - docker run -it --rm -d -p 5000:5000 max-image-caption-generator
+  - docker build -f "$DOCKERFILE" -t max-image-caption-generator:"$IMAGE"-"$ARCH"-"$VERSION" .
```

Comment:
This is why Travis is failing
`The command "docker build -f "$DOCKERFILE" -t max-image-caption-generator:"$IMAGE"-"$ARCH"-"$VERSION" ." failed and exited with 125 during .`

https://travis-ci.org/IBM/MAX-Image-Caption-Generator/jobs/565642780

Reply:
It looks like the IMAGE/ARCH variables aren't being populated. I'll mirror this to an internal repo and see if I can figure out what's going on.

Reply:
You are defining the variables in the matrix below but they are undefined here